src/rci.js

```diff
@@ -52,6 +52,15 @@ const collector = {
   WebVitalsCollector
 };
 
+/** Trigger an custom event to signal that the CORE was loaded successfully */
+/** Registering the new event constructor */
+const eventName = 'RCICoreReady';
+const RCICoreReadyEvent = new Event(eventName);
+
+/** Dispatch the event and assign the event as a property for non event driven approach*/
+window.dispatchEvent(RCICoreReadyEvent);
```

Comment:
@robertdumitrescu INFO - I'll assume that dispatching this event before export won't have any negative side-effect (race condition) due to the asynchronous nature of JS..?

Reply:
Fixed